tianshou/data/buffer.py

```diff
@@ -115,13 +115,16 @@ class ReplayBuffer:
         than or equal to 1, defaults to 1 (no stacking).
     :param bool ignore_obs_next: whether to store obs_next, defaults to
         ``False``.
+    :param bool save_last_obs: only save the last obs/obs_next when it has a
+        shape of (timestep, ...), defaults to ``False``.
```

Comment:
```suggestion
    :param bool save_last_obs: only save the last obs/obs_next when it has a
        shape of (timestep, ...) because of temporal stacking, defaults to ``False``.
```